Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -6396,42 +6396,58 @@ def com_google_fonts_check_metadata_category_hint(family_metadata):
     """,
-    proposal = 'https://github.com/googlefonts/fontbakery/issues/3886'
+    proposal = ['https://github.com/googlefonts/fontbakery/issues/3886',
+                'https://github.com/googlefonts/fontbakery/issues/4131',
+                'https://googlefonts.github.io/gf-guide/color.html']
 )
 def com_google_fonts_check_colorfont_tables(ttFont):
-    """Check font has the expected color font tables"""
+    """Check font has the expected color font tables."""
+    passed = True
+
     if "COLR" in ttFont:
-        colr_table = ttFont["COLR"]
-        if colr_table.version == 0 and "SVG " in ttFont:
-            yield FAIL, Message(
-                "drop-svg",
-                "Font has a COLR v0 table, which is already widely supported, "
-                "so the SVG table isn't needed."
-            )
-            return
-        elif colr_table.version == 1 and "SVG " not in ttFont:
-            yield FAIL, Message(
-                "add-svg",
-                "Font has COLRv1 but no SVG table; for CORLv1, we require "
-                "that an SVG table is present to support environments where "
-                "the former is not supported yet."
-            )
-            return
-    elif "SVG " in ttFont:
+        if ttFont["COLR"].version == 0 and "SVG " in ttFont:
+            passed = False
+            yield FAIL,\
+                  Message("drop-svg",
+                          "Font has a COLR v0 table, which is already widely supported,"
+                          " so the SVG table isn't needed.")
+
+        elif ttFont["COLR"].version == 1 and "SVG " not in ttFont:
+            passed = False
+            yield FAIL,\
+                  Message("add-svg",
+                          "Font has COLRv1 but no SVG table; for CORLv1, we require"
+                          " that an SVG table is present to support environments where"
+                          " the former is not supported yet.\n"
+                          "You can do it by using the maximum_color tool provided by"
+                          " the Nano Emoji project:\n"
+                          "https://github.com/googlefonts/nanoemoji")
+
+    if "SVG " in ttFont:
+        if is_variable_font(ttFont):
+            passed = False
+            yield FAIL,\
+                  Message("variable-svg",
+                          "This is a variable font and SVG does not support"
+                          " OpenType Variations.\n"
+                          "Please remove the SVG table from this font.")
+
         if "COLR" not in ttFont:
-            yield FAIL, Message(
-                "add-colr",
-                "Font only has an SVG table. Please add a COLR table as well. "
-            )
-            return
-    yield PASS, "Looks Good!"
+            passed = False
+            yield FAIL,\
+                  Message("add-colr",
+                          "Font only has an SVG table."
+                          " Please add a COLR table as well.")
```

Comment:
you could suggest to use the same maximum_color tool like you did for the `add-svg` message above. The tool can also generate a COLRv1 table from SVG table

Reply:
Great!